public/canjs/controls/select-issue-type/select-issue-type.js

```diff
@@ -2,9 +2,8 @@ import { StacheElement, type, ObservableObject, ObservableArray, value, queues }
 
 import {updateUrlParam, pushStateObservable} from "../../routing/state-storage.js";
 import { bitoviTrainingIssueData } from "../../../examples/bitovi-training.js";
-import { getSimplifiedIssueHierarchy } from "../../../stateful-data/jira-data-requests.js";
-import { mostCommonElement } from "../../../utils/array/array-helpers.js";
 
+import routeData, { issueHierarchyFromNormalizedIssues } from "../../routing/route-data.js";
```

Comment:
in the TR-209 PR, issueHierarchyFromNormalizedIssues and other functions that may be shared between route data and local data are moved to a utils library.